internal/handlers/common/distinct.go

```diff
@@ -43,6 +43,7 @@ type DistinctParams struct {
 	Collation *types.Document `ferretdb:"collation,unimplemented"`
 
 	ReadConcern *types.Document `ferretdb:"readConcern,ignored"`
+	LSID        any             `ferretdb:"lsid,ignored"`
```

Comment:
We want to specify type instead of `any`. Should it be `*types.Document`?  https://www.mongodb.com/docs/manual/reference/server-sessions/#command-options

This applies to all other place :hugs:

Reply:
Sounds good!

Reply:
I think it's OK to leave the type as `any` because it could be a top level field in certain situations.